### PR TITLE
added dom pseudo element test

### PR DESCRIPTION
**Pull request by chaim-aharonson-applitools**

### CODE REVIEW

File: DotNet/overrides.js [hunk 1]
```diff
@@ -41,4 +41,5 @@ module.exports = {
     // Skipped test running on Selenium3 but not on Selenium4
     "should send dom on edge legacy": {skip: true},
     "Should return exception in TestResultsSummary": {skipEmit: true},
+    'should get dom pseudo elements when check window': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline DotNet/overrides.js:44]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "should get dom pseudo elements when check window": {skipEmit: true},
```

File: python/overrides.js [hunk 2]
```diff
@@ -5,6 +5,7 @@
     // Stale element are not handled by python binding
     'should handle check of stale element if selector is preserved': {skip: true},
     'should handle check of stale element in frame if selector is preserved': {skip: true},
-
-    "Should return exception in TestResultsSummary": {skipEmit: true},
+    
+    'Should return exception in TestResultsSummary': {skipEmit: true},
+    'should get dom pseudo elements when check window': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline python/overrides.js:10]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "should get dom pseudo elements when check window": {skipEmit: true},
```

File: java/overrides.js [hunk 3]
```diff
@@ -60,7 +60,8 @@
     "appium android landscape mode check region on android 7": {skipEmit: true},
     "appium android landscape mode check window on android 10": {skipEmit: true},
     "appium android landscape mode check window on android 7": {skipEmit: true},
-    "should work with beforeCaptureScreenshot hook": {skip: true}
+    "should work with beforeCaptureScreenshot hook": {skip: true }, 
+    'should get dom pseudo elements when check window': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline java/overrides.js:64]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "should get dom pseudo elements when check window": {skipEmit: true},
```

File: ruby/overrides.js [hunk 4]
```diff
@@ -19,5 +19,6 @@
     'should return browserInfo in getAllTestResults': {skip: true},
     'should be empty if page delayed by 1500': {skip: true},
 
-    "Should return exception in TestResultsSummary": {skipEmit: true},
+    'Should return exception in TestResultsSummary': {skipEmit: true},
+    'should get dom pseudo elements when check window': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline ruby/overrides.js:23]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "should get dom pseudo elements when check window": {skipEmit: true},
```

File: python/overrides.js [hunk 5]
```diff
@@ -5,6 +5,7 @@
     // Stale element are not handled by python binding
     'should handle check of stale element if selector is preserved': {skip: true},
     'should handle check of stale element in frame if selector is preserved': {skip: true},
-
-    "Should return exception in TestResultsSummary": {skipEmit: true},
+    
+    'Should return exception in TestResultsSummary': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline python/overrides.js:9]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "Should return exception in TestResultsSummary": {skipEmit: true},
```

File: ruby/overrides.js [hunk 6]
```diff
@@ -19,5 +19,6 @@
     'should return browserInfo in getAllTestResults': {skip: true},
     'should be empty if page delayed by 1500': {skip: true},
 
-    "Should return exception in TestResultsSummary": {skipEmit: true},
+    'Should return exception in TestResultsSummary': {skipEmit: true},
```
Comment (Yarden-Ingber) [inline ruby/overrides.js:22]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9515763)
```suggestion
    "Should return exception in TestResultsSummary": {skipEmit: true},
```

File: coverage-tests.js [hunk 7]
```diff
@@ -1261,6 +1261,25 @@ test('should send dom and location when check window', {
   }
 })
 
+test('should get dom pseudo elements when check window', {
```
Comment (amitzur) [inline coverage-tests.js:1264]: Honestly, I don't think we need a generic coverage test for this fix. Only a test in `dom-capture`. I can explain this to anyone if needed.

**Review by amitzur — CHANGES_REQUESTED** [review]
I believe this is unnecessary.

File: coverage-tests.js (hunk 7 above)
Comment (chaim-aharonson-applitools) [reply coverage-tests.js:1264]: test was removed (actually the whole PR)

**Pull request closed without merging**